resources/howtos/search_bar_for_docs.md

```diff
@@ -100,10 +101,9 @@ If you don't want to use Docker, here are [other ways to run the scraper](https:
 
 :::
 
-`<your-meilisearch-host-url>` and `<your-meilisearch-api-key>` should be replaced by the credentials of the MeiliSearch instance you set up at [the first step](./search_bar_for_docs.md#run-a-meilisearch-instance).
 `<absolute-path-to-your-config-file>` should be the **absolute** path of your configuration file defined at [the previous step](./search_bar_for_docs.md#configuration-file).
 
-The API key you must provide should have the permissions to add documents into your MeiliSearch instance. Thus, you need to provide the private key.
+The API key you must provide should have the permissions to add documents into your MeiliSearch instance. It means the private key is enough to perform this kind of request. Thus, in a production environment, we recommend providing the private key instead of the master key.
```

Comment:
```suggestion
The API key you must provide should have the permissions to add documents into your MeiliSearch instance. In a production environment, we recommend providing the private key.
```

Reply:
Your suggestion seems not clear to me. I would rather (at least) let "instead of the master key" to continue with the main example.

Reply:
Updated ;)

Reply:
And why don't you want to let the explanation about the private key is enough? ^^

Reply:
Is not that I don't want the explanation, I think it's the formulation that I don't like. Maybe just removing the "it means". I updated my suggestion again.